notebooks/official/tensorboard/vertex_tensorboard_custom_training_with_prebuilt_container.ipynb

```diff
@@ -0,0 +1,856 @@
+{
```

Comment:
<p>No need to have "vertex" in the file name. Please see the naming recommendations here:&nbsp;<a href="http://go/cloudai-notebook-workflow" target="_blank">go/cloudai-notebook-workflow</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/603/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Done</p>